dev/lsp/src/features/completion.ts

```diff
@@ -1,13 +1,65 @@
-import { TextDocumentPositionParams } from "vscode-languageserver";
+import {
+  TextDocumentPositionParams,
+  CompletionList,
+  CompletionItem,
+  CompletionItemKind,
+  Range,
+  Position,
+  CompletionItemLabelDetails,
+} from "vscode-languageserver";
+import { URI } from "vscode-uri";
 import { LSPInstance } from "../lsp";
+import { 
+    getBuiltinsDoc, 
+    LobsterBuiltinsDoc, 
+    LobsterFunctionSignature 
+} from "../lobster";
+
+let builtinsDoc: LobsterBuiltinsDoc[];
+
+export function getFunctionSignature(
```

Comment:
This shouldn't be in a feature since its not only for this feature. Features are meant to be fully independent of eachother.